docs/source/en/model_doc/bloom.mdx

```diff
@@ -25,6 +25,13 @@ Several smaller versions of the models have been trained on the same dataset. BL
 - [bloom-7b1](https://huggingface.co/bigscience/bloom-7b1)
 - [bloom](https://huggingface.co/bigscience/bloom) (176B parameters)
 
+## Languages
+
+BLOOM has officially been trained in 46 languages. The full list of trained languages are:
```

Comment:
Probably 

BLOOM has officially been trained in 46 human languages and X programming languages.

I am not sure if `officially` is necessary.

Reply:
Great! Should be addressed in ed1e74972899d3f05c6886ea61c6f517c96f734e